fs/fs.go

```diff
@@ -35,7 +35,7 @@ import (
 	"github.com/golang/glog"
 )
 
-var partitionRegex = regexp.MustCompile("^(:?(:?s|xv)d[a-z]+\\d*|dm-\\d+)$")
+var partitionRegex = regexp.MustCompile(`^(:?(:?s|xv)d[a-z]+\d*|dm-\d+)$`)
```

Comment:
Does this regex optionally start with `::`, or is that supposed to be a non-capturing group? If the latter, it's backwards:
`(?:re)`

Also would prefer to move this closer to where it's used.